src/main/java/com/sparta/popupstore/domain/common/exception/ErrorCode.java

```diff
@@ -10,13 +10,16 @@ public enum ErrorCode {
     // company error
 
     // popupStore error
-
+    POPUPSTORE_NOT_FOUND(HttpStatus.BAD_REQUEST,"해당 팝업스토어가 없습니다."),
```

Comment:
```suggestion
    POPUPSTORE_NOT_FOUND(HttpStatus.NOT_FOUND,"해당 팝업스토어가 없습니다."),
```
BAD_REQUEST를 사용한 이유가 없다면 NOT_FOUND로 해주세요.